### PR TITLE
Improve Code Quality (Pass 1)

### PR DESCRIPTION
**Pull request by WhiredPlanck**
#### Issue tracker
Fixes will automatically close the related issues

Fixes # N/A

#### Feature
Describe features of this pull request

See the commit records.

#### Code of conduct
- [x] [CONTRIBUTING](CONTRIBUTING.md)

#### Style lint
- [x] `make sytle-lint`

#### Build pass
- [x] `make debug`

#### Manually test
- [x] Done

#### Code Review
1. No wildcards import
2. Manual build and test pass
3. GitHub Action CI pass
4. At least one contributor review and approve
5. Merged clean without conflicts
6. PR will be merged by rebase upstream base

#### Daily build
Login and download artifact at https://github.com/osfans/trime/actions

#### Additional Info

N/A

### CODE REVIEW

File: app/src/main/java/com/osfans/trime/ime/symbol/SimpleAdapter.kt [hunk 1]
```diff
@@ -71,7 +63,14 @@ class SimpleAdapter(theme: Theme, private val columnSize: Int) : RecyclerView.Ad
...
                 this.ellipsize = TextUtils.TruncateAt.MARQUEE
-                this.background = mBackground
+                this.background =
+                    ColorManager.getDrawable(
+                        context,
+                        "key_back_color",
+                        "key_border",
+                        "key_border_color",
+                        "round_corner",
+                    )
```
Comment (goofyz) [inline app/src/main/java/com/osfans/trime/ime/symbol/SimpleAdapter.kt]: 這樣改的話，每個 `viewHolder` 都會生成一個 `drawable`，效能上比之前會差了一點點。

File: app/src/main/java/com/osfans/trime/data/theme/ColorManager.kt [hunk 2]
```diff
@@ -331,16 +331,17 @@ object ColorManager {
 
     //  返回图片或背景的drawable,支持null参数。 Config 2.0
     fun getDrawable(
-        key: String?,
-        borderKey: String?,
-        borderColorKey: String?,
-        roundCornerKey: String,
-        alphaKey: String?,
+        context: Context,
```
Comment (goofyz) [inline app/src/main/java/com/osfans/trime/data/theme/ColorManager.kt]: 加入 `context` 就是為了計 `dp2px`，似乎有點不化算。
將來建議先在外面計算 border 大小，再用來呼叫這個 function 。

**Review by goofyz — APPROVED** [review]
Pass 2 應該要刪除掉大部份的 debug log，現在 logging 實在太多了。

File: app/src/main/java/com/osfans/trime/ime/symbol/SimpleAdapter.kt (hunk 1 above)
Comment (WhiredPlanck) [reply app/src/main/java/com/osfans/trime/ime/symbol/SimpleAdapter.kt]: 是个问题，我想想可以怎么弄 ……

File: app/src/main/java/com/osfans/trime/data/theme/ColorManager.kt (hunk 2 above)
Comment (WhiredPlanck) [reply app/src/main/java/com/osfans/trime/data/theme/ColorManager.kt]: 我再调整看看

**Pull request merged**